islandora_videojs.module

```diff
@@ -64,9 +66,12 @@ function template_preprocess_islandora_videojs(array &$variables) {
   if (isset($params['sources']) && is_array($params['sources'])) {
     $variables['sources'] = $params['sources'];
   }
-  else {
+  elseif (isset($params['url']) && isset($params['mime'])) {
     $variables['sources'][] = array('url' => $params['url'], 'mime' => $params['mime']);
   }
+  else {
+    $variables['sources'] = array();
```

Comment:
So this works nicely to remove the warnings, but you are left with what appears to be a working player but nothing plays. Maybe we should add a warning like.
```
drupal_set_message("The video file is missing from this object. Please contact the site administrator.", "warning");
```
into this else statement?